pyttxt.py

```diff
@@ -0,0 +1,14 @@
+bcainfo=""" BCA course info:
+it is 3 years undergraduate course."""
+with open("BCA.txt", "w") as file:
+     file.write(bcainfo)
+rows={'x':0,'y':0, 'z':0}
+with open('BCA.txt','r') as bca_intro:
+    lines=bca_intro.readlines()
+    for line in lines:
+            f_char=line.strip()[0]
+            if f_char in rows:
+                rows[f_char]+=1

```

Comment:
explain how to work for loop?

Reply:
1.The init step is executed first, and only once. This step allows you to declare and initialize any loop control variables. ...
2.Next, the condition is evaluated. ...
3.After the body of the 'for' loop executes, the flow of control jumps back up to the increment statement. ...
4.The condition is now evaluated again.